src/routes/DigitalAssets/routes/Send/sagas/digitalAssetsSend.js

```diff
@@ -89,116 +88,209 @@ function* openView(action: ExtractReturn<typeof digitalAssetsSend.openView>): Sa
   }
 }
 
-function* getAmountError(amount: string, digitalAsset: ?DigitalAsset): Saga<?string> {
-  const isValidAmount: boolean = (parseFloat(amount) > 0)
+function* requestGasPrice(): Saga<?BigNumber> {
+  const network: ExtractReturn<typeof selectCurrentNetwork> = yield select(selectCurrentNetwork)
 
-  if (!isValidAmount) {
-    return 'Amount should be greater than 0'
+  if (!network) {
+    throw new Error('There is no active network')
   }
 
-  if (!digitalAsset) {
+  try {
+    const gasPrice: BigNumber = yield call(web3.getGasPrice, network)
+    return gasPrice
+  } catch (err) {
```

Comment:
it we catch exception from this call and user will not set gasPrice value manually what is happen?

Reply:
https://github.com/jibrelnetwork/jwallet-web/pull/507/files#diff-3b2aeb1c694883811531669ea4f8266eR145

User will get the error 'cant fetch gasprice'
Then he can try to send transaction again by press send button